test/integration/targets/reboot/tasks/main.yml

```diff
@@ -1,39 +1,58 @@
-- name: Test reboot
-  when: ansible_facts.virtualization_type | default('') not in ['docker', 'container', 'containerd']
-  block:
-    # This block can be removed once we have a mechanism in ansible-test to separate
-    # the control node from the managed node.
-    - block:
-        - name: Write temp file for sanity checking this is not the controller
-          copy:
-            content: 'I am the control node'
-            dest: /tmp/Anything-Nutlike-Nuzzle-Plow-Overdue
-          delegate_to: localhost
-          connection: local
-          when: inventory_hostname == ansible_play_hosts[0]
+- name: Memorize whether we're in a containerized environment
+  set_fact:
+    in_container_env: >-
+      {{
+        (
+          (
+            ansible_facts.virtualization_type
+            | default('')
+          )
+          in ['docker', 'container', 'containerd']
+        )
+        | bool
+      }}
+
```

Comment:
Combine this with the `set_fact` task below.

Reply:
This is done.